docs/DeveloperGuide.md

```diff
@@ -540,17 +437,14 @@ The application does not offer functionality for contacting tutees.
 **UC06: View a specific tutee**
 
 **MSS**
-1. User requests to list tutees.
+1. User requests to list tutees. (UC04)
```

Comment:
For inclusion of use cases, perhaps we can use formatting to style it as such the textbook suggests, i.e:
User requests to <ins>list tutees. (UC04)</ins>